packages/flutter/test/services/text_formatter_test.dart

```diff
@@ -6,7 +6,18 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+class TestTextInputFormatter extends TextInputFormatter {
+  const TestTextInputFormatter();
+}
+
 void main() {
+  test('test const constructor', () {
+    const testValue1 = TestTextInputFormatter();
+    const testValue2 = TestTextInputFormatter();
+
+    expect(testValue1, equals(testValue2));
```

Comment:
```suggestion
    expect(testValue1, same(testValue2));
```

https://pub.dev/documentation/matcher/latest/matcher/same.html